### PR TITLE
[PT-BR] Update `Featured_Artists`

### PR DESCRIPTION
**Pull request by GranularBow8193**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Featured_Artists/pt-br.md [hunk 1]
```diff
@@ -1,17 +1,41 @@
...
-outdated_since: 84df5a5c33bb4256a47c830bc67b6f3bb7636846
+tags:
+  - FA
+  - Artistas em destaque
 ---
 
 # Artistas em destaque
 
-**Artistas em destaque** são músicos que trabalharam junto ao osu! para fazer uma seleção de suas próprias músicas disponíveis para uso irrestrito na comunidade. Alguns desses artistas em destaque também comporam músicas específicamente para o osu! e lançaram elas como exclusivas do osu!
+**Artistas em destaque** são musicas de artistas que trabalharam com o osu! para fazer uma seleção de suas faixas ficarem disponíveis para uso em [beatmaps](/wiki/Beatmap) e imagem de fundo em músicas de transmissões ao vivo da [Copa do mundo osu!](/wiki/Tournaments#osu!-world-cup).
```
Comment (LeoFLT) [inline wiki/Featured_Artists/pt-br.md:9]: Any reason for this particular change? The old wording is more accurate

Comment (GranularBow8193) [reply wiki/Featured_Artists/pt-br.md:9]: not really, i forgot the old one and translated without verifying it

i'm going to pay attention about that

File: wiki/Featured_Artists/pt-br.md [hunk 2]
```diff
@@ -1,17 +1,41 @@
...
+Adições recentes para a biblioteca de artistas em destaque são selecionados indiretamente pela comunidade do osu!. Se mapeadores criarem e ranquearem seus mapas para um artista, o [osu! team](/wiki/People/The_Team) pode comunicar com o artista para discutir o Programa de Artistas em Destaque. Porém, alguns artistas são ocasionalmente convidados para de tornar um Artista em Destaque baseado em outros fatores, assim como sua expectativa de contato com a comunidade de mapeamento do osu!.
 
-Mais informações sobre a Mappers' Guild pode ser encontradas em [seu website](http://mappersguild.com) e no canal `#mappers-guild` no [Servidor de Discord do osu!dev](https://discord.gg/ppy).
+A maioria das musicas no programa de Artistas em Destaque não exclusivamente licenciadas para o osu!. Em casos raros um artista irá criar faixas exclusivamente para o osu!, que é caracterizado como *exclusivo do osu!* em sua listagem de Artista em Destaque.
```
Comment (LeoFLT) [inline wiki/Featured_Artists/pt-br.md:17]:
```suggestion
A maioria das musicas no programa de Artistas em Destaque licenciadas para o osu! sem exclusividade. Em casos raros um artista irá criar faixas exclusivamente para o osu!, que é caracterizado como *exclusivo do osu!* em sua listagem de Artista em Destaque.
```

**Review by LeoFLT — CHANGES_REQUESTED** [review]
Looks good to me besides this one

**Review by LeoFLT — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Review by Walavouchey — CHANGES_REQUESTED** [review]
you forgot one paragraph at the bottom

> If a pre-timed .osz from the Featured Artist listing has errors of any kind, they can be reported on [this forum thread](https://osu.ppy.sh/community/forums/topics/783985).

File: wiki/Featured_Artists/pt-br.md [hunk 3]
```diff
@@ -1,17 +1,41 @@
 ---
-no_native_review: true
-outdated: true
-outdated_since: 84df5a5c33bb4256a47c830bc67b6f3bb7636846
+tags:
+  - FA
+  - Artistas em destaque
```
Comment (Walavouchey) [inline wiki/Featured_Artists/pt-br.md:4]: already in the title

```suggestion
```

**Comment by GranularBow8193** [comment]
i accidentally removed the review, sorry

**Comment by Walavouchey** [comment]
ah, no, that's supposed to happen

File: wiki/Featured_Artists/pt-br.md [hunk 4]
```diff
@@ -39,3 +38,5 @@ Mapeadores são encorajados a criar mapas com faixas de Artistas em Destaque. Es
 Todos os concursos de mapeamento publicados na [listagem de concursos da comunidade](https://osu.ppy.sh/community/contests), assim como os [Concursos de Mapeamento Mensais](/wiki/Contests/Monthly_Beatmapping_Contest) e a [Olimpíada de Mapeamento](/wiki/Contests/Mapping_Olympiad), precisam que participantes façam beatmaps de músicas de Artistas em Destaque. Alguns concursos feitos pela comunidade utilizam essas músicas também.
 
 Para promover o conteúdo de Artistas em Destaque, membros da comunidade criaram o [Mappers' Guild](/wiki/Community/Mappers_Guild), dando jogadores um sistema de desafio com estilo MMO que eles podem participar com outros mapeadores. Prêmios são distribuídos por criar e ranquear um beatmap, incluindo 3 tipos de medalhas de perfil<!-- TODO: profile badges article -->. Mapeadores podem entrar na guilda visitando o [site de Mappers' Guild](https://mappersguild.com/).
+
+Se um pré-tenporizado `.osz` pela listagem de Artista em Destaque possuir erros de qualquer tipo, eles podem ser reportados [nesse fórum](https://osu.ppy.sh/community/forums/topics/783985).
```
Comment (LeoFLT) [inline wiki/Featured_Artists/pt-br.md:42]:
```suggestion
Se um `.osz` pré-temporizado da listagem de Artistas em Destaque possuir erros de qualquer tipo, eles podem ser reportados [nesse fórum](https://osu.ppy.sh/community/forums/topics/783985). 
```

**Review by LeoFLT — CHANGES_REQUESTED** [review]
Good to merge after this little grammatical change

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**